request/models.py

```diff
@@ -21,6 +21,7 @@ class Request(models.Model):
     method = models.CharField(_('method'), default='GET', max_length=7)
     path = models.CharField(_('path'), max_length=255)
     time = models.DateTimeField(_('time'), default=timezone.now, db_index=True)
+    response_time = models.FloatField(_('response_time'), null=True)
```

Comment:
This field is being added under the request comment, perhaps it should be moved to line 18 under response.

Reply:
Sounds good